R/get_uuid.R

```diff
@@ -43,6 +43,20 @@ get_uuid <- function(name = NULL, n = 1, url = FALSE) {
   # API call -------------------------------------------------------------
   opts <- list()
   if (!is.null(name)) opts$filter_name <- name
```

Comment:
I looked into it, and it looks like the first node returned without any `filter_name` is Pan-biota, so this still works for uses where users want to get a lot of random phylopics. 👍